docs/sources/k6/next/testing-guides/calculate-concurrent-users.md

```diff
@@ -0,0 +1,101 @@
+---
+title: 'Calculate concurrent users for load tests'
+description: 'Learn how to use Google Analytics data to determine the optimal number of concurrent users for your load tests.'
+weight: 200
+---
+
+# Calculate concurrent users for load tests
+
+When planning load tests, determining the right number of concurrent users is crucial for creating realistic test scenarios. This guide explains how to use Google Analytics data to calculate concurrent user numbers and design effective load tests.
+
+{{< admonition type="note" >}}
+
+You can use any analytics tool that provides the same metrics to help you calculate the concurrent users you need for load testing.
+
+{{< /admonition >}}
+
+## Understanding concurrent users
+
+Google Analytics tracks visitors ("Users") and their activity periods ("Sessions") on your site. During a session, users generate traffic through page loads or AJAX requests that your servers must handle. To create realistic load tests, you need to determine how many users are actively generating traffic at any given time.
+
+The recommended formula for calculating your baseline concurrent users is:
+
+```
+Concurrent users  = Hourly sessions * Average session duration (in seconds) / 3600
+```
+
+## Get the data from Google Analytics
+
+To find the required metrics in Google Analytics:
+
+- Log in to your Google Analytics account.
+- Navigate to the **Reporting** tab.
+- Click **Audience** on the sidebar menu.
+- Click **Overview**.
+- Set your desired time period.
+- Click **Hourly** view in the line graph settings.
+
+You should be able to view the session count and average session duration on the overview dashboard.
+
+{{< admonition type="note" >}}
+
+These instructions might vary depending on which version of Google Analytics you're using. If you're using Google Analytics 4, you can find these metrics by selecting **Explore** and creating a **Free form** report.
+
+{{< /admonition >}}
+
+## Designing load tests
```

Comment:
We explain here why it doesn't make sense to have bigger windows but what about smaller? For example, why doesn't make sense to have by minutes or seconds?